src/chainstate/stacks/index/mod.rs

```diff
@@ -163,19 +169,19 @@ impl From<MARFValue> for BlockHeaderHash {
     }
 }
 
-impl From<MARFValue> for u64 {
-    fn from(m: MARFValue) -> u64 {
+impl From<MARFValue> for u32 {
+    fn from(m: MARFValue) -> u32 {
         let h = m.0;
-        let mut d = [0u8; 8];
-        for i in 0..8 {
+        let mut d = [0u8; 4];
+        for i in 0..4 {
             d[i] = h[i];
         }
-        for i in 8..h.len() {
+        for i in 4..h.len() {
             if h[i] != 0 {
-                panic!("Failed to convert MARF value into u64: data stored after 8th byte");
+                panic!("Failed to convert MARF value into u64: data stored after 4th byte");
```

Comment:
Glad to see this panics as expected, per my earlier comment.  However, the warning should read `u32` not `u64` ;)

Reply:
Addressed in d31f0207